Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Dockerfile
+# https://medium.com/@wkrzywiec/build-and-run-angular-application-in-a-docker-container-b65dbbc50be8
+# Stage 1:  Build 
+
+FROM node:12.7-alpine AS build
+
+WORKDIR /usr/src/app
+
+COPY package.json ./
+
+RUN npm install
```

Comment:
let's copy package-lock.json also, and then run `npm ci` instead.

Reply:
We added these two commands and figured out that our problem was we placed the Dockerfile and other files in the wrong place (they were up one directory too high). So we think that the other comment you had has also been addressed correctly. As of right now, I was able to build and run the Dockerized app locally so I think this is ready to merge. The last thing we were unsure of was the diff to package-lock but as long as everyone uses the same Dockerfile(which means using the same node version), it should be correct I believe.

Reply:
@dannydjg16 if it's ready for re-review then mark this conversation as resolved and re-request review from me in the top right

Reply:
it's not ready though cause there's a huge diff to package-lock which has nothing to do with the purpose of this PR.
if we're going to standardize on a version of npm that uses lockfileversion 1, then i could let this through, but i'd better not see a massive diff changing the file back in another PR. get some other people to run `npm i` and see if they get lockfileversion 1 or 2 at the top of package-lock. whichever is more common, everyone else get on the version of node/npm that those people have.

Reply:
cc @yetanothersegfault

Reply:
We have removed the package-lock.json from this PR. I am going to ensure that we have consistency moving forwards by ensuring that everyone is using npm 7.x to make us all in line with the original scaffold